cunumeric/runtime.py

```diff
@@ -277,6 +280,62 @@ def create_wrapped_scalar(
         )
         return DeferredArray(self, store, dtype=dtype)
 
+    def bitgenerator_populate_task(
+        self, task, taskop, generatorID, generatorType=0, seed=0, flags=0
+    ):
+        task.add_scalar_arg(taskop, ty.int32)
+        task.add_scalar_arg(generatorID, ty.int32)
+        task.add_scalar_arg(generatorType, ty.uint32)
+        task.add_scalar_arg(seed, ty.uint64)
+        task.add_scalar_arg(flags, ty.uint32)
+
+    def bitgenerator_create(
+        self, generatorType, seed, flags, forceCreate=False
+    ):
+        self.current_random_bitgenid = self.current_random_bitgenid + 1
+        if forceCreate:
```

Comment:
do you have any test that exercises this path? otherwise this code path can be bitrotted

Reply:
test_force_build